lib/manager/common.ts

```diff
@@ -161,6 +161,7 @@ export interface Upgrade<T = Record<string, any>>
   toVersion?: string;
   updateType?: string;
   version?: string;
+  nestedVersion?: boolean;
```

Comment:
Where does the need for this come from? I thought it should always be a part of `managerData`

Reply:
Let me check this for the exact details. As I remember, only one function was passing `nestedVersion` value, and I added it to the interface.